### ISSUE TITLE
Unable to set fileExt

### ISSUE THREAD

**Issue by macmichael01** [mjmlio/gulp-mjml#53]
I am trying to set `fileExt` to `.email` but the files are still getting output as `.html` After digging through the code, I found that [this line](https://github.com/mjmlio/gulp-mjml/blob/71c6d20b84487876aa27f750dd5a5b0c9116111d/src/index.js#L50) is reflecting a different value. Instead, what I am seeing is: `output.path = replaceExt(file.path.toString(), ".html");`

Has the npm package been updated to reflect the latest changes?

**Comment by pabueco** [comment]
Same problem here. I temporarely fixed it by using [gulp-rename](https://www.npmjs.com/package/gulp-rename):
```
.pipe(rename(path => {
    path.extname = '.extension';
}))
```

**Comment by macmichael01** [comment]
npm is not in sync with what is on github. Please bump npm version.

**Comment by iRyusa** [comment]
I'll see if I can do it this week

**Comment by Zaszczyk** [comment]
> I'll see if I can do it this week

Have you tried to fix it? I have same problem.

**Comment by iRyusa** [comment]
I think this has been shipped for a while now ? 3.0.1 has it

**Comment by Zaszczyk** [comment]
> I think this has been shipped for a while now ? 3.0.1 has it

Right, but it does not work. Process finish successfully, but files keep old extension.

**Comment by iRyusa** [comment]
Can you share maybe your current gulp file ?

Edit : it seems like current npm version doesn't have the fileExt options I will update it asap
